recipes/xpack/all/conanfile.py

```diff
@@ -0,0 +1,28 @@
+from conans import ConanFile, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class XpackConan(ConanFile):
+    name = "xpack"
+    description = "C++ reflection, ability to convert between C++ structures and json/xml."
+    topics = ("xpack", "json", "reflection", "xml")
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/xyz347/xpack"
+    license = "Apache-2.0"
+    no_copy_source = True
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version], strip_root=True, destination=self._source_subfolder)
+
+    def package(self):
+        self.copy(pattern="LICENSE", dst="licenses", src=self._source_subfolder)
+        self.copy(pattern="*", dst=os.path.join("include", "xpack"), excludes=["example/*", "gtest/*"], src=self._source_subfolder)
```

Comment:
not sufficient, packaging is not very neat:

```
xpack/1.0.2 package(): Packaged 54 '.h' files
xpack/1.0.2 package(): Packaged 3 files: .gitignore, LICENSE, LICENSE
xpack/1.0.2 package(): Packaged 1 '.pri' file: xpack.pri
xpack/1.0.2 package(): Packaged 1 '.md' file: README.md
xpack/1.0.2 package(): Packaged 4 '.hpp' files: rapidxml_print.hpp, rapidxml_iterators.hpp, rapidxml.hpp, rapidxml_utils.hpp
xpack/1.0.2 package(): Packaged 1 '.html' file: manual.html
xpack/1.0.2 package(): Packaged 1 '.txt' file: license.txt
```

```suggestion
        self.copy(pattern="*.h", dst=os.path.join("include", "xpack"), excludes=["example/*", "gtest/*", "thirdparty/*"], src=self._source_subfolder)
        # TODO: remove bundled dependencies
        self.copy(pattern="*", dst=os.path.join("include", "xpack", "thirdparty"), src=os.path.join(self._source_subfolder, "thirdparty"))
```